src/utility/timer.ts

```diff
@@ -1,6 +1,6 @@
-let timer;
+let timer: number | NodeJS.Timeout | undefined;
```

Comment:
I don't think it's a `NodeJS.Timeout` as it will be running client-side (in the browser) and so it's just a number or undefined